imap_processing/swe/tests/test_swe_l1a_event_msg.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+import pytest
+
+from imap_processing import imap_module_directory
+from imap_processing.swe.l0 import decom_swe
+
+
+@pytest.fixture(scope="session")
+def decom_test_data():
+    """Read test data from file"""
+    test_folder_path = "swe/tests/l0_data"
+    event_file_list = list(
+        imap_module_directory.glob(f"{test_folder_path}/*SWE_EVTMSG_packet.bin")
+    )
+    event_data = []
+    for packet_file in event_file_list:
+        event_data.append(decom_swe.decom_packets(packet_file)[0])
+
+    return event_data
```

Comment:
Is this comment accurate? I wasn't entirely clear why you were doing this in just code at first.
```suggestion
    # Return a list of the first packet from every file
    return [decom_swe.decom_packets(packet_file)[0] for packet_file in event_file_list]
```

Reply:
no. I got event data in multiple files. And each file contains one event data only. In this for loop, I am going through all event data file and reading those one data and return as one for testing.